masq_lib/src/test_utils/utils.rs

```diff
@@ -3,8 +3,11 @@
 use std::fs;
 use std::path::PathBuf;
 
-pub const DEFAULT_CHAIN_ID: u8 = 3u8; //For testing only
-pub const TEST_DEFAULT_CHAIN_NAME: &str = "ropsten"; //For testing only
+pub const TEST_DEFAULT_CHAIN_ID: u8 = 2;
```

Comment:
This is inconsistent with the Ethereum chain numbers. 
mainnet = 1
undefined = 2
ropsten = 3
rinkeby = 4
etc.

I strongly encourage that we stick with and not deviate from the standard network numbers.
<img width="593" alt="EthereumNetworkNumbers" src="https://user-images.githubusercontent.com/60554948/135737204-683a7202-7b5e-495f-8094-a3a586a1504c.png">